### PR TITLE
Allow `@aragon/templates-shared` to be published

### PR DESCRIPTION
**Pull request by sohkai**
Adds npm configuration to `shared/` so we can publish it :).

### CODE REVIEW

File: shared/package.json [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
   "description": "Shared helpers for Aragon templates",
+  "author": "Aragon Association <legal@aragon.org>",
+  "license": "GPL-3.0-or-later",
+  "files": [
+    "contracts/",
+    "ethereum-data",
+    "helpers/",
+    "lib/",
+    "scripts/",
+    ".env",
```
Comment (sohkai) [inline shared/package.json]: I'm not really sure I understand the need for the `.env` file. I see we use `ETHDATA_LOCAL_PATH` in the docker-compose file, but does that get automatically injected by docker?

Comment (facuspagnuolo) [reply shared/package.json]: We shouldn't add it here definitely. It is used by `dotenv` which we already dropped, so we don't need it here.
I don't see why adding `ethereum-data` and `docker-compose.yml`, these completely custom to our testing task.

Comment (sohkai) [reply shared/package.json]: But we're sharing the testing utils with others; do you think it'd be useful if they had these geth-helpers too (like we do in our templates)?

Comment (facuspagnuolo) [reply shared/package.json]: Now you're asking, yea, it's weird... maybe it is being injected by docker as you were saying

**Pull request merged**